deploy/cloudflare.md

```diff
@@ -12,28 +12,38 @@ outline: deep
 Uploading via Web UI is limited to 300 MB.
 
 Use [rclone](https://rclone.org/downloads/) to upload larger PMTiles archives to R2.
+In order to use rclone you'll need to create an API key, and copy the "Access Key ID", "Secret Access Key" and the "Endpoints for S3 clients"
+running rclone can be done using docker so you wouldn't need to install anything locally.
```

Comment:
Rclone is a zero-install single binary so I recommend that in the docs - I think mentioning Docker here is superfluous and may confuse readers, instead just show the commands using plain `rclone` and those that prefer to use Docker to run rclone can extrapolate.

Reply:
Sure, I've updated the instructions to only mention the docker image as the download website or rclone doesn't list it.

Reply:
Are we certain that `rclone/rclone` in the default namespace (Docker hub) comes from the rclone authors? Seems like https://registry.hub.docker.com/r/rclone/rclone would be the place to check but that doesn't load for me...

Reply:
I believe we are, see:
https://github.com/rclone/rclone/blob/8503282a5adffc992e1834eed2cd8aeca57c01dd/.github/workflows/build_publish_release_docker_image.yml#L39

Reply:
No clue why this isn't mentioned anywhere in the docs...